.github/workflows/pro-extension-test.yml

```diff
@@ -182,12 +182,12 @@ jobs:
       - name: Get Artifact ID
         working-directory: ${{ inputs.artifactPath }}
         id: get-artifact-id
```

Comment:
as GitHub-hosted windows runners include a Git Bash environment,we can use shell: bash  as Jake mentioned. It wouldn't interfere with already existing repos running the same workflow.
eg : adding `shell: bash`